# 1.Question
1.Suppose a child psychologist says that the average time that working mothers spend talking to their children is at-least 11 minutes per day. You want to test You conduct a random sample of 1000 working mothers and find they spend an average of 11.5 minutes per day talking with their children. Assume prior research suggests the population standard deviation is 2.3 minutes. Conduct test with a level of significance of alpha=0.05

#Defining Null and alternate hypotheses

Null hypothesis => The average time that working mothers spend talking to their children is at-least 11 minutes per day. 
Alternate hypothesis => The average time that working mothers spend talking to their children is less than 11 minutes per day.

Choose Significance Level

Given level of significance of alpha as 0.05

Identifying a statistical test

From the given data it is understood that the datasize is high and Standard deviation is given. So z test can be used.

Calculating the test statistic

In [3]:
z=(11.5-11)/2.3  
z

0.2173913043478261

# Comparing it with the level of significance

So,alternative hypothesis contains the less sign , 
Left-tailed test has to be performed. In that,comparison of the test statistic value and critical value is done. From the z table the critical value obtained is -1.645.since test statistic(0.217) is greater than critical value(-1.645), 
hence we can accept the null hypothesis.

# Conclusion — rejecting or not rejecting your null hypothesis

It is concluded with the null hypothesis is acceptable,i.e., there is enough evidence to support the claim that " The average time that working mothers spend talking to their children is at-least 11 minutes per day".

# 2.Question

A coffee shop claims that their average wait time for customers is less than 5 minutes. To test this claim, a sample of 40 customers is taken, and their wait times are recorded. The sample mean wait time is found to be 4.6 minutes with a standard deviation of 0.8 minutes. Perform a hypothesis test at a significance level of 0.05 and determine whether there is enough evidence to support the coffee shop's claim. Perform the hypothesis testing using correct test statistics and appropriate critical value

# Null and Alternate Hypothesis

Null hypothesis => The average wait time for customers is less than 5 minutes. Alternate hypothesis => The average wait time for customers is greater than 5 minutes.

#Choose Significance Level

Its already given that level of significance of alpha=0.05

Identifying a statistical test

From the given details we know the sample size is small(40) and the population standard deviation is not known. So here we can use t test.

Calculating the test statistic





In [5]:
t=(4.6-5)/(0.8/(40**0.5))
t

-3.162277660168382

# Comparing it with the level of significance

With reference to test statistic , right-tailed test has to be performed. In that,comparison of the test statistic value and critical value is done. From the z table the critical value obtained is -1.6849.
since test statistic(-3.16) is less than critical value(-1.645), hence we can accept the null hypothesis.
 
Test statistic is less than critical value so our null hypothesis is in the acceptence region.

Conclusion
Now there is enough evidence to support the claim that "The average wait time for customers is less than 5 minutes" .